### ISSUE TITLE
Move chirped CMSSW IOSite fields into a single field

### ISSUE THREAD

**Issue by stiegerb** [dmwm/cms-htcondor-es#82]
Rather than the existing format, that creates at least N(sitename) field names:

```python
{
   "ChirpCMSSW(.*?)IOSite_Site1_ReadBytes" : 123456,
   "ChirpCMSSW(.*?)IOSite_Site1_ReadTimeMS" : 123456,
   "ChirpCMSSW(.*?)IOSite_Site2_ReadBytes" : 234567,
   "ChirpCMSSW(.*?)IOSite_Site2_ReadTimeMS" : 234567
}
```

we should store these in a single nested field:

```python
{"ChirpCMSSW_IOSite" : [
   {
      "SiteName" : "Site1",
      "ReadBytes" : 123456,
      "ReadTimeMS" : 123456
   },
   {
      "SiteName" : "Site2",
      "ReadBytes" : 234567,
      "ReadTimeMS" : 234567
   }
]}
```

Aggregation by site name can still work when using a kibana plugin: https://ppadovani.github.io/knql_plugin/overview/

I will add this field, leaving the old fields in place for now. Then we can set up the plugin, test it, and decide how to proceed. @bbockelm what do you think? Do you know who's currently using this information?

**Comment by bbockelm** [comment]
Hm - it's not clear to me, what's the advantage here?  Does it lessen the load on the database or is it purely sugarcoating things?

**Comment by stiegerb** [comment]
I think from a performance point of view we're still ok with a few hundred field names. But that can easily change, since whatever produces these is clearly broken (and should also be fixed). So if we don't pay attention the number might explode.

The general issue is that we're putting information in keys that should be stored in values. So we can't search for it, we can't group by it, and we don't even know what they're called most of the time. So those fields are **only** useful if you know exactly which one you need (and you trust that none of "T2_US_UCSD" gets actually filled in "3947291384" instead, otherwise it will be incomplete).

Changing the format, and using that plugin, we should at least be able to make a plot of data volume grouped by sitename, and get an idea of what's going on.

What's your use case currently for these?

**Comment by stiegerb** [comment]
One more reason that I just discovered:
`root:ERROR - Failed to index 7 documents to ES: [(u'Limit of total fields [1000] in index [cms-2019-01-24] has been exceeded', 7)]`

I guess we should ask the ES and MONIT teams to increase that number.

**Comment by bbockelm** [comment]
Ah - that in itself is a good reason.  Let’s go with it!

**Comment by stiegerb** [comment]
Merged #86, so I'm closing this.
